petrinet/examples/sir.json

```diff
@@ -8,6 +8,7 @@
       {
         "id": "S",
         "name": "Susceptible",
+        "description": "Number of individuals relative to the total population that are 'susceptible' to a disease infection",
```

Comment:
In #17 I added units that assume absolute counts for this example model. Could you remove the "relative to the total population" part of the description?

Reply:
Done, I also removed it for `typed_sir.json` in anticipation!